### PR TITLE
Week 1: Pizza-project Alma Herrström

### PR DESCRIPTION
**Pull request by almaherris**
## Netlify link

https://transcendent-rabanadas-a06927.netlify.app/

## Collaborators

[almaherris]

### CODE REVIEW

File: .gitignore
Comment (AntonellaMorittu) [inline .gitignore:1]: .DS_Store is a file that stores custom attributes of its containing folder, such as folder view options, icon positions, and other visual information, and we don't need to share it in our project. Please add .DS_Store to your .gitignore file (if you don't have one, you need to create it) 👍

File: code/script.js
Comment (AntonellaMorittu) [inline code/script.js:1]: Please format your code to have the right spaces and indentation 👍

File: code/switch.js [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+    `Welcome to our JavaScript Pizzeria. Ready to order? - Click 'OK' to begin.`
+  )
+  
+  let userName = prompt(
+    "What is your name?"
+    )
+  
+  switch (userName) {
+    case "":
+        alert(`We didn't catch that, please refresh the page and try again!`)
```
Comment (AntonellaMorittu) [inline code/switch.js:15]: nice touch validating the name input! 💯

File: code/switch.js
Comment (AntonellaMorittu) [inline code/switch.js:1]: Please format your code to have the right spaces and indentation 👍

File: code/switch.js [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+  let age = ""
+  let ageInput = parseInt ( 
+    prompt (
+    `Is this order for a child or an adult? 
+Please enter your age:`
+  )
+  )
+
+  switch (true) {
+    case ageInput >= 16 && ageInput <= 120:
```
Comment (AntonellaMorittu) [inline code/switch.js:159]: great you put an age limit!

File: pull_request_template.md
Comment (AntonellaMorittu) [inline pull_request_template.md:1]: Please delete all files that are not useful for the application, like `pull_request_template.md`. That's there for you to copy the content and use it in the PR, after that can be deleted 🧹

**Review by AntonellaMorittu — APPROVED** [review]
Hi Alma, well done with the Pizzeria project! 🍕  You worked really well, the app has a good flow and the code meets the requirements and stretch goals, that's great! 

I left some comments specifically on some parts, please take a look at them :) Thank you ⭐

**Comment by almaherris** [comment]
Hi Antonella! Thanks for your comments - I deleted the pull request template and formatted the code using Prettier. When it comes to the DS file, I had already created a gitignore file for it, so I don't understand how that is not reflected on your side.. 🤨

**Comment by AntonellaMorittu** [comment]
hi Alma, great! I don't see the DS file you're right, my bad, sorry about that 🥹

**Pull request closed without merging**